### PR TITLE
🧪 Add filippo-fonseca to data.json

### PR DESCRIPTION
**Pull request by filippo-fonseca**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -455,4 +455,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png",
     "issueNumber": 237
   }
```
Comment (AllanRegush) [inline data.json:457]: Please add a comma here
```suggestion
  },
```

File: data.json [hunk 2]
```diff
@@ -455,4 +455,10 @@
...
+  {
+    "name": "Filippo Fonseca",
+    "githubUsername": "filippo-fonseca",
+    "imageUrl": "https://user-images.githubusercontent.com/48065878/104632919-feb5a400-5663-11eb-8dd0-b05cfd377158.png",
+    "issueNumber": 244
+}
```
Comment (AllanRegush) [inline data.json:463]: Could you format the Brace here like the rest?

**Review by AllanRegush — COMMENTED** [review]
Thank You for your PR 👍 Looking good. Just a few adjustments.

**Comment by filippo-fonseca** [comment]
> Thank You for your PR 👍 Looking good. Just a few adjustments.

Done! Let me know if I need to change anything. :)

**Review by AllanRegush — APPROVED** [review]
Looks Good 👍

**Pull request merged**